src/uct/cuda/cuda_ipc/cuda_ipc_iface.c

```diff
@@ -200,11 +200,9 @@ static ucs_status_t uct_cuda_ipc_iface_query(uct_iface_h tl_iface,
     iface_attr->device_addr_len         = sizeof(uint64_t);
     iface_attr->ep_addr_len             = 0;
     iface_attr->max_conn_priv           = 0;
-    iface_attr->cap.flags               = UCT_IFACE_FLAG_ERRHANDLE_PEER_FAILURE |
```

Comment:
i think removing `UCT_IFACE_FLAG_ERRHANDLE_PEER_FAILURE` cap means cuda_ipc can't be used for rndv whith client-server apps. Did you check it is still selected?